src/View/Admin/Includes/_formulary_course.html.twig

```diff
@@ -0,0 +1,60 @@
+<div class="formulary-course-admin col-md-8 offset-md-2 mt-5">
+    {% if errors %}
+        <ul class="alert alert-danger">
+            {% for error in errors %}
+                <li>{{error}}</li>
```

Comment:
good practice is to put spaces after the 2 opening curly braces and before the 2 closing curly braces in twig

Reply:
correction done